connect/api/src/main/java/org/apache/kafka/connect/sink/SinkConnectorContext.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.connect.sink;
+
+import org.apache.kafka.connect.connector.ConnectorContext;
+
+/**
+ * SinkConnectorContext is provided to SinkConnector to allow them to interact with the underlying
+ * runtime.
+ *
+ * This interface can be used to add specifics methods to a SinkConnector.
```

Comment:
I'd suggest a description that's a bit more straightforward and more closely mirrors the JavaDoc in the `ConnectorContext`:

> A context to allow a {@link SinkConnector} to interact with the Kafka Connect runtime.